packages/manager/apps/web/client/app/email-domain/email/redirection/email-domain-email-redirection.controller.js

```diff
@@ -61,9 +71,19 @@ angular.module('App').controller(
       this.loading.redirections = true;
       this.redirections = null;
 
-      return this.WucEmails.getRedirections(this.$stateParams.productId)
-        .then((data) => {
-          this.redirections = data.map((id) => ({ id }));
+      return this.iceberg('/email/domain/:domain/redirection')
+        .query()
+        .expand('CachedObjectList-Pages')
+        .execute({
+          domain: this.$stateParams.productId,
+        })
+        .$promise.then((result) => {
+          this.redirections = result.data.sort((a, b) => {
+            if (a.from === b.from) {
+              return a.to > b.to ? 1 : -1;
+            }
+            return a.from > b.from ? 1 : -1;
+          });
```

Comment:
this sort logic is used in multiple places. This can be reused by extracting it into new method